lib/logs.js

```diff
@@ -55,14 +55,16 @@ module.exports = {
 			orderBy: 'LastEventTime',
 		};
 
+		let aliasGetAliasFunctionVersion
+		// Check if --version is specified
+		if (this._options.version) {
+			aliasGetAliasFunctionVersion = BbPromise.resolve(this._options.version)
```

Comment:
There are some missing semicolons. Please check the Travis build (https://travis-ci.org/HyperBrain/serverless-aws-alias/builds/325347600) for details.